build-tools/packages/build-tools/src/common/monoRepo.ts

```diff
@@ -91,6 +91,7 @@ export class MonoRepo {
 		private readonly logger: Logger = defaultLogger,
 	) {
 		this.version = "";
+		this.workspaceGlobs = [];
 		const pnpmWorkspace = path.join(repoPath, "pnpm-workspace.yaml");
 		const lernaPath = path.join(repoPath, "lerna.json");
 		const yarnLockPath = path.join(repoPath, "yarn.lock");
```

Comment:
we only have one use of yarn in our repo and its not a workspace: do we need to support it here?

Reply:
Technically, no, but I have some ongoing experiments with different package managers and this eases testing. It's not critical to keep if you feel strongly about it, but I'd do it separately anyway.